src/tutorials/mobile/index.md

```diff
@@ -2,7 +2,11 @@
 
 [[toc]]
 
-In the last tutorial we created a new <MainPlatformName /> project in QGIS with layers for surveying trees and hedges. You'll now learn how to get fixme.
+In the last tutorial we created a new <MainPlatformName /> project in QGIS with layers for surveying trees and hedges. You'll now learn how to:
+* Open the QGIS project in <MobileAppName />
+* Set GPS accuracy thresholds

```

Comment:
not sure if thresholds or "threshold" is grammatically correct?

Reply:
It should be correct, I checked with the merriam webster dictionary and also found some similar uses online.